docs/source/reference/terminator.rst

```diff
@@ -18,4 +18,11 @@ The :mod:`~optuna.terminator` module implements a mechanism for automatically te
    optuna.terminator.CrossValidationErrorEvaluator
    optuna.terminator.StaticErrorEvaluator
    optuna.terminator.TerminatorCallback
-   optuna.terminator.report_cross_validation_scores
\ No newline at end of file
+   optuna.terminator.report_cross_validation_scores
+
+For an example of using the `optuna.terminator` module, please refer to `this link <https://github.com/optuna/optuna-examples/tree/main/terminator>`_.
+
+.. seealso::
+   :ref:`terminator_example` tutorial provides an example of using the `optuna.terminator` module.
+
+.. _terminator_example: https://github.com/optuna/optuna-examples/tree/main/terminator
```

Comment:
> Thank you for updating the PR. I'm sorry that my comment was not right because the linked page is not tutorial. So the following sentence would be enough.

NO problem, I have again made changes as suggested by you, please review it once.